src/Controller/Backend/FileManager.php

```diff
@@ -330,12 +331,13 @@ private function getRelatedFiles(FileInterface $file)
     {
         // Match foo(_local).*(.dist)
         $base = $file->getFilename();
-        if (Str::endsWith($base, '.dist')) {
+        Path::getExtension($base);
+        if (Path::getExtension($base) === '.dist') {
             $base = substr($base, 0, -5);
         }
         $ext = pathinfo($base, PATHINFO_EXTENSION);
-        $base = Str::replaceLast(".$ext", '', $base);
-        $base = Str::replaceLast('_local', '', $base);
+        $base = Str::replaceLast($base, ".$ext", '');
+        $base = Str::replaceLast($base, '_local', '');
```

Comment:
Done but tweaked as the `.dist` doesn't make sense here, they're not installed as `.dist` in end-user installs, don't list the other two (normal & _local) when being edited, and if they're available in the editor why would they be able to be saved? (they're distribution files, not end-user ones)